README.md

```diff
@@ -412,4 +412,4 @@ var movement = new Movement({
 
 As we see, we can easily get data from multiple sources, transform it on the fly (degrees to radians and mph to m/s) and compose it into a new model. Notice though that `Movement` does not in any way depend on either a compass or a chip log. If you're not on a boat, you may wish to procure your data by different means.
 
-To explain in a bit more detail what's happening with each of the external fields, let's examine the `heading`. What this does is add a `heading : coconut.data.Value<Float>` to the constructor argument's fields. The `Value<T>` is a helper, that will directly consume any `Observable<T>` but will wrap any other expression in [`Observable.auto`](https://github.com/haxetink/tink_state#automagic-observables). The observable is consumed by the constructor and is directly exposed, making it opaque to the outside world whether the very origin of the data is *within* the model or external to it.
\ No newline at end of file
+To explain in a bit more detail what's happening with each of the external fields, let's examine the `heading`. What this does is add a `heading : coconut.data.Value<Float>` to the constructor argument's fields. The `Value<T>` is a helper, that will directly consume any `Observable<T>` but will wrap any other expression in [`Observable.auto`](https://github.com/haxetink/tink_state#automagic-observables). The observable is consumed by the constructor and is directly exposed, making it opaque to the outside world whether the very origin of the data is *within* the model or external to it.
```

Comment:
Nothing - I think the github editor might have removed the last newline character resulting in this vague diff?